### DISCUSSION TITLE
How do I setup the FallingTree mod on my Minecraft server?

### DISCUSSION THREAD

**Discussion by theinsidersandrush** [RakambdaOrg/FallingTree#394] [Q&A] [answered]
Hi. I am trying to find the answer about how to setup FallingTree mod on my Minecraft server.

**Comment by Rakambda** [comment, marked as answer]
Like any other mods, drop the mod file (jar file) in the mods folder of a forge/fabric server.

**Reply by theinsidersandrush** [reply]
But I use Paper/SpigotMC for my server.

**Reply by Rakambda** [reply]
Paper/Spigot uses plugins, which is different from mods. You need a forge or Fabric server.
